src/domain/community/contributor/CommunityContributorsBlockWide/CommunityVirtualContributorsBlockWide.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box, Theme, useMediaQuery } from '@mui/material';
+import PageContentBlock from '../../../../core/ui/content/PageContentBlock';
+import PageContentBlockHeader from '../../../../core/ui/content/PageContentBlockHeader';
+import GridItem from '../../../../core/ui/grid/GridItem';
+import GridProvider from '../../../../core/ui/grid/GridProvider';
+import Gutters from '../../../../core/ui/grid/Gutters';
+import MultipleSelect from '../../../../core/ui/search/MultipleSelect';
+import ContributorCardSquare from '../ContributorCardSquare/ContributorCardSquare';
+import { useState } from 'react';
+import { useColumns } from '../../../../core/ui/grid/GridContext';
+import { VirtualContributorProps } from '../../community/VirtualContributorsBlock/VirtualContributorsDialog';
+import { ContributorType } from './CommunityContributorsBlockWideContent';
+import Loading from '../../../../core/ui/loading/Loading';
+import usePlatformOrigin from '../../../platform/routes/usePlatformOrigin';
+
+interface CommunityContributorsBlockWideProps {
+  virtualContributors: VirtualContributorProps[];
+  isLoading?: boolean;
+}
+
+const COMPACT_VIEW_ITEMS_LIMIT = 3 * 8;
+
+const CommunityVirtualContributorsBlockWide = ({
+  virtualContributors,
+  isLoading,
+}: CommunityContributorsBlockWideProps) => {
+  const [filter, onFilterChange] = useState<string[]>([]);
+  const isSmallScreen = useMediaQuery<Theme>(theme => theme.breakpoints.down('lg'));
+  const columns = useColumns();
+
+  const filterFn = (filter: string[]) => (element: VirtualContributorProps) => {
+    return (
+      filter.length === 0 ||
+      filter.some(
+        term =>
+          element.profile.displayName.toLowerCase().includes(term.toLowerCase()) ||
+          element.profile.tagline.toLowerCase().includes(term.toLowerCase())
```

Comment:
Again the filtering by tagline seems redundant. 

Also, the `filterFn` could be better named because the following is not pretty:
.filter(filterFn(filter))

It would look a bit better if it's `virtualContributors.filter(matchesNameFilter(searchTerm))` or something like that.